tensorflow/inference/docker/2.2.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,105 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI" 
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+
+ARG PYTHON=python3.7
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.7.7
+ARG OPENSSL_VERSION=1.1.1g
+ARG TFS_SHORT_VERSION=2.2
+ARG TFS_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/serving/r2.2_aws/20200606-011120/cpu/py37/tensorflow_model_server
+
+# See http://bugs.python.org/issue19846
+ENV LANG=C.UTF-8
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+ENV LD_LIBRARY_PATH='/usr/local/lib:$LD_LIBRARY_PATH'
+ENV MODEL_BASE_PATH=/models
+# The only required piece is the model name in order to differentiate endpoints
+ENV MODEL_NAME=model
+ENV DEBIAN_FRONTEND=noninteractive
+
+# nginx + njs 
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    curl \
+    gnupg2 \
+    ca-certificates \
+    git \
+    wget \
+    vim \
+    build-essential \
+    zlib1g-dev \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \ 
+ && echo 'deb http://nginx.org/packages/ubuntu/ bionic nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends \
+    nginx \
+    nginx-module-njs \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN ${PIP} --no-cache-dir install --upgrade pip setuptools
+
+# cython, falcon, gunicorn, grpc
+RUN ${PIP} install --no-cache-dir \
+    awscli \
+    cython==0.29.14 \
+    falcon==2.0.0 \
+    gunicorn==20.0.4 \
+    gevent==1.4.0 \
+    requests==2.22.0 \
+    grpcio==1.27.1 \
+    protobuf==3.11.1 \
+# using --no-dependencies to avoid installing tensorflow binary
+ && ${PIP} install --no-dependencies --no-cache-dir \
+    tensorflow-serving-api==2.2.0
+
+COPY ./sagemaker /sagemaker
+
+# put tensorflow library (with only error_codes) to python dist-packages
+WORKDIR /
+RUN cd /usr/local/lib/python3.*/dist-packages/ \
+ && mv /sagemaker/tensorflow-2.2 ./tensorflow \
+# Delete the remaining tensorflow folder to avoid confusing python import
+ && rm -rf /sagemaker/tensorflow
+
+# Some TF tools expect a "python" binary
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
```

Comment:
Since the arguments does not match, this step is not taking place and making the security check fail as there is no python path(in addition to python 37) available